substrate/bin/node/runtime/src/migrations.rs

```diff
@@ -21,14 +24,44 @@ pub mod add_contracts_registry_pallet {
 		}
 
 		fn on_runtime_upgrade() -> Weight {
-			let mut weight = DbWeight::get().reads(1);
-
-			if StorageVersion::get::<ContractsRegistry>() == 0 {
-                StorageVersion::new(1).put::<ContractsRegistry>();
-                weight = weight.saturating_add(DbWeight::get().reads_writes(1, 1));
-            }
-
-            weight
+			let identities = vec![
+				("5EYCAe5hvejUE1BUTDSnxDfCqVkADRicSKqbcJrduV1KCDmk", b"Vault".to_vec()),
+				("5EYCAe5hveooUENA5d7dwq3caqM4LLBzktNumMKmhNRXu4JE", b"Senate".to_vec()),
+				(
+					"5EYCAe5iXF2YZuVZv1vig4xvf1CcDVocZCWYrv3TVSXpMTYA",
+					b"Citizenship Office".to_vec(),
+				),
+				//("5EYCAe5ijiYfyeZ2JJCGq56LmPyNRAKzpG4QkoQkkQNB5e6Z", b"Treasury"), - skipping this as Polkadot.js Apps have it builtin
+				("5EYCAe5g8CDuMsTief7QBxfvzDFEfws6ueXTUhsbx5V81nGH", b"Congress".to_vec()),
+				(
+					"5CSxW2nn4mQckisBziai4wGqQLEdKTUA4Xnt8iy8jtj5q52Q",
+					b"Onboarder LLD Faucet".to_vec(),
```

Comment:
onboarder faucet has the identity already, skip it

Add the offices though, company registry office, land and metaverse land offices, and the upcoming ministry of finance office

Reply:
Done.